cfgov/legacy/templates/complaint/complaint-landing.html

```diff
@@ -35,13 +35,147 @@
+              </span>
+            </p>
+            <p>Have an issue with a financial product or service? We have
+              <a href="/ask-cfpb/">
+                answers to frequently asked questions
+              </a>
+              and can help you connect with companies if you have a complaint.
+              <a href="/complaint/">
+                Learn more
+              </a>
+            </p>
+            <h2 id="what-you-should-consider">What you should consider when using the data</h2>
+            <p>This database is not a statistical sample of consumers’ experiences in the marketplace. Complaints are not necessarily representative of all consumers’ experiences and complaints do not constitute “information” for purposes of the
+              <a href="/open-government/information-quality-guidelines/">
+                Information Quality Act
+              </a>.
+            </p>
+            <p>Complaint volume should be considered in the context of company size and/or market share. For example, companies with more customers may have more complaints than companies with fewer customers. We encourage you to pair complaint data with public and private data sets for additional context.</p>
+            <p>The Bureau publishes the consumer’s narrative description of his or her experience if the consumer opts to share it publicly and after the Bureau takes steps to remove personal information. We don’t verify all the allegations in complaint narratives. Unproven allegations in consumer narratives should be regarded as opinion, not fact. We do not adopt the views expressed and make no representation that consumers’ allegations are accurate, clear, complete, or unbiased in substance or presentation. Users should consider what conclusions may be fairly drawn from complaints alone.</p>
+            <h2>How we use complaint data</h2>
+            <p>Complaints can give us insights into problems people are experiencing in the marketplace and help us regulate consumer financial products and services under existing federal consumer financial laws, enforce those laws judiciously, and educate and empower consumers to make informed financial decisions. We also report on complaint trends annually in
+              <a href="/data-research/research-reports/2018-consumer-response-annual-report/">
+                Consumer Response’s Annual Report to Congress
+              </a>.
+            </p>
+            <p>
+              <a href="/complaint/data-use/">
+                More on how we collect and use the data
+              </a>
+            </p>
+          </div>
+          <div class="content-l_col content-l_col-1-3 db-numbers callouts-large">
+            <p class="callout"><span class="numeric-callout percent-timely">97%</span> of complaints sent to companies get timely responses</p>
+          </div>
+        </div>
+      </div>
+      <div class="block block__bg ccdb-landing_section block__flush-top block__flush-bottom">
```

Comment:
Maybe reorder the classes so all the block classes are together.

Reply:
Fixed